contrib/babelfishpg_money/fixeddecimal.c

```diff
@@ -2981,7 +2981,10 @@ fixeddecimalaggstateserialize(PG_FUNCTION_ARGS)
 	if (!AggCheckCallContext(fcinfo, NULL))
 		elog(ERROR, "aggregate function called in non-aggregate context");
 
-	state = (FixedDecimalAggState *) PG_GETARG_POINTER(0);
+	state = PG_ARGISNULL(0) ? NULL : (FixedDecimalAggState *) PG_GETARG_POINTER(0);
+
+	if (state == NULL)
```

Comment:
Or simply we can have `if (PG_ARGISNULL(0)) return NULL` before line 2984

Reply:
done